chat-sdk-ui/src/main/java/co/chatsdk/ui/chat/ChatActivity.java

```diff
@@ -85,6 +85,7 @@
     // If we are showing a temporary screen like the sticker message screen
     // this should be set to no
     protected boolean removeUserFromChatOnExit = true;
+    protected boolean addUserToChatOnEnter = true;
```

Comment:
This decides if a user should be added to a thread just by looking into a thread rather than being invited by a thread admin. This was important for the Alex List project so that only the admin can add users to a thread which then would receive push notifications

Reply:
But this is the behaviour which Alex wanted as far as I remember. The user would still be able to see the messages but only a member added by an admin would receive push notifications. Do you have an alternative idea on how to implement this?